Src/Json/Shared.Json/ObjectDiffPatch.cs

```diff
@@ -0,0 +1,378 @@
+﻿#region *   License     *
+/*
+    SimpleHelpers - ObjectDiffPatch   
```

Comment:
So finally i don't see a quick & clean way to use the structural equivalency API for `JToken` - which may be due to my lack of understanding ;-)

My current suggestion is to reduce the risk of maintenance by cutting down the `ObjectDiffPatch` code down to the bare minimum needed for FA.